rest/config_test.go

```diff
@@ -2619,3 +2599,15 @@ func TestCollectionsValidation(t *testing.T) {
 		})
 	}
 }
+
+func assertX509UnknownAuthority(t testing.TB, actual, expected error) {
```

Comment:
I'd rename to `require` to match the underlying behaviour (for context: `require.` stops test execution at the failure point, `assert.` continues running the test code but will still record it as a failed test)

```suggestion
func requireX509UnknownAuthority(t testing.TB, actual, expected error) {
```